templates/compose/swetrix.yaml

```
@@ -0,0 +1,165 @@
+# documentation: https://docs.swetrix.com/selfhosting/how-to
+# slogan: Privacy-friendly and cookieless European web analytics alternative to Google Analytics.
+# category: analytics
+# tags: analytics,privacy,monitoring,open-source,clickhouse,redis
+# logo: svgs/swetrix.svg
+# port: 3000
+
+services:
+  swetrix:
+    image: swetrix/swetrix-fe:v4.0.2
+    restart: always
+    depends_on:
+      - swetrix-api
+    environment:
+      # Required - public URL of the API service (e.g. https://swetrix-api.example.com)
+      - API_URL=${API_URL:?}
+    healthcheck:
+      test:
+        - CMD-SHELL
+        - wget --no-verbose --tries=1 --spider http://localhost:3000/ping || exit 1
+      interval: 30s
+      start_period: 15s
```

Adding timeout and retries would be helpful

```suggestion
      timeout: 5s
      retries: 10
      interval: 30s
      start_period: 15s
```